### DISCUSSION TITLE
Referencing specific SVG file

### DISCUSSION THREAD

**Discussion by rtambun** [gapitio/gapit-htmlgraphics-panel#165] [Q&A] [answered]
Hello, 

I would like to implement below panel

![image](https://github.com/gapitio/gapit-htmlgraphics-panel/assets/20750135/9a65366f-b02a-42ec-b082-8191bb9f00fb)

How to reference to SVG file that will be representing downward (or upward arrow) ? The only example i can see is builtin shape like circle or box.

Thanks

**Comment by ZuperZee** [comment, marked as answer]
You would have to copy the contents of the SVG file to the HTML/SVG section of the panel.
![image](https://github.com/gapitio/gapit-htmlgraphics-panel/assets/39485579/9d3c5c63-a457-40f5-a913-e3c092edc894)

**Reply by ZuperZee** [reply]
If you are new, I would suggest you check some examples on the website.
https://gapit-htmlgraphics-panel.gapit.io/docs/examples/change-svg-attributes-with-grafana-variables-example/
https://gapit-htmlgraphics-panel.gapit.io/docs/quick-start/
https://gapit-htmlgraphics-panel.gapit.io/docs/guides/how-to-get-metrics/
